### PR TITLE
Show the latest run tag in the run list table

### LINKED ISSUE #958: Show the latest run tag in the run list table
Show the latest run tag in the run list table

### PR DESCRIPTION
**Pull request by csordasmarton**
Closes #958

### CODE REVIEW

File: api/v6/report_server.thrift [hunk 1]
```diff
@@ -116,7 +116,8 @@ struct RunData {
   4: i64                       duration,     // Duration of the run (-1 if not finished).
   5: i64                       resultCount,  // Number of results in the run.
   6: string                    runCmd,       // The used check command.
-  7: map<DetectionStatus, i32> detectionStatusCount // Number of reports with a particular detection status.
+  7: map<DetectionStatus, i32> detectionStatusCount, // Number of reports with a particular detection status.
```
Comment (gyorb) [inline api/v6/report_server.thrift]: This is an api change, so api version should be increased accordingly.

Comment (csordasmarton) [reply api/v6/report_server.thrift]: I increased the version in these files:
- libcodechecker/version.py
- www/scripts/version.js

Do I have to change the api version in other files too?

File: api/v6/report_server.thrift [hunk 2]
```diff
@@ -116,7 +116,8 @@ struct RunData {
...
+  8: string                    versionTag    // Version tag of the latest run.
```
Comment (whisperity) [inline api/v6/report_server.thrift]: ❗️ ❗️ Please be **very careful**about changing the fields of a `struct`. Thrift supports an innate "versioning" for adding new member fields. Please check **extensively** if an older client can still parse this new, modified version of the struct!

Comment (csordasmarton) [reply api/v6/report_server.thrift]: I tested it with a 6.0 client by calling the `CodeChecker cmd runs`

File: libcodechecker/server/client_db_access_handler.py [hunk 3]
```diff
@@ -529,7 +529,19 @@ def getRunData(self, run_filter):
...
 
-            q = session.query(Run, stmt.c.report_count)
+            tag_q = session.query(RunHistory.run_id,
+                                  func.max(RunHistory.version_tag).label(
+                                      'run_history_tag'),
+                                  func.max(RunHistory.time).label(
+                                      'run_history_time')) \
+                .filter(RunHistory.version_tag.isnot(None)) \
+                .group_by(RunHistory.run_id) \
+                .order_by('run_history_time') \
```
Comment (bruntib) [inline libcodechecker/server/client_db_access_handler.py]: I think this ordering is not necessary.

File: api/v6/report_server.thrift (hunk 1 above)
Comment (gyorb) [reply api/v6/report_server.thrift]: These two files should be enough.

**Comment by csordasmarton** [comment]
Yes, I have fixed it.

**Pull request merged**